databuilder/models/table_metadata.py

```diff
@@ -59,18 +59,86 @@ def create_next_relation(self):
             return None
 
 
+class DescriptionMetadata:
+    DESCRIPTION_NODE_LABEL = 'Description'
+    PROGRAMMATIC_DESCRIPTION_NODE_LABEL = 'Programmatic_Description'
+    DESCRIPTION_KEY_FORMAT = '{description}'
+    DESCRIPTION_TEXT = 'description'
+    DESCRIPTION_SOURCE = 'description_source'
+
+    DESCRIPTION_RELATION_TYPE = 'DESCRIPTION'
+    INVERSE_DESCRIPTION_RELATION_TYPE = 'DESCRIPTION_OF'
+
+    # The default editable source.
+    DEFAULT_SOURCE = "description"
+
+    def __init__(self,
+                 source,  # type: str
```

Comment:
Should we assign default value to source as non-programmatic?
`source=DEFAULT_SOURCE`

Seems it currently default to programmatic description label where source value can be still `None`.

Reply:
will do